pylint/checkers/utils.py

```diff
@@ -842,3 +842,41 @@ def is_registered_in_singledispatch_function(node):
             return decorated_with(func_def, singledispatch_qnames)
 
     return False
+
+
+def splitlines(value, keepends=False):
+    """Splitlines only on universal newlines for consistency across python2
```

Comment:
Not just a py2/3 issue: in python 3 `str.splitlines` splits on a superset of universal newlines, while `bytes.splitlines` splits on universal newlines.

Without a change here, a formfeed will raise 'missing-final-newline' in python3, so if you don't fix #1218 then there is no reason to fix #1219 since it will never get past #1218.

The other solution would be to keep the builtin `str.splitlines` and check for the superset characters and recombine them with the next line.